docs/buildtest_tutorial_examples/spack/inspect/spack_env_deactivate.txt

```diff
@@ -0,0 +1,31 @@
+Error File: /home/spack/runs/generic.local.bash/spack_env_deactivate/spack_env_deactivate_first/756844a5/spack_env_deactivate_first.err
+Log File: /home/spack/buildtest/var/logs/buildtest_r1__w6f3.log
+──────────────── Test File: /home/spack/runs/generic.local.bash/spack_env_deactivate/spack_env_deactivate_first/756844a5/spack_env_deactivate_first.sh ────────────────
+#!/bin/bash                                                                                                                                                            
+source /home/spack/spack/share/spack/setup-env.sh                                                                                                                      
+spack env create  -d /home/spack/spack-envs/m4                                                                                                                         
+spack env activate  -d /home/spack/spack-envs/m4                                                                                                                       
+spack add m4                                                                                                                                                           
+spack install                                                                                                                                                          
+                                                                                                                                                                       
+                                                                                                                                                                       
+######## START OF POST COMMANDS ########                                                                                                                               
+spack find                                                                                                                                                             
+rm -rf $HOME/spack-envs/m4                                                                                                                                             
+                                                                                                                                                                       
+######## END OF POST COMMANDS   ########                                                                                                                               
+                                                                                                                                                                       
+                                                                                                                                                                       
```

Comment:
Please rerun the example here because we need to see the line `spack env deactivate` in the generated test. At the moment your implementation needs to be fixed first